ci/docker/docker-compose.yml

```diff
@@ -108,6 +108,16 @@ services:
         BASE_IMAGE: nvidia/cuda:10.1-cudnn7-devel-ubuntu18.04
       cache_from:
         - ${DOCKER_CACHE_REGISTRY}/build.ubuntu_gpu_cu101:latest
+  ubuntu_gpu_cu102:
```

Comment:
Let's delete the ubuntu_gpu_cu101 then?

Reply:
I recommended serge to add instead of update. I wasn't sure if we should migrate all the existing versions from cu101 to cu102. Do we want to test a few things on cu101 as we continue to support?
Or since this is master [proxy for 2.0], do we move to the cuda 10.2 & then potentially to 11?

Reply:
It's safer to move only this test on 10.2 for now, I agreed with @ChaiBapchya

Reply:
Generally we can't test all possible cuda versions and thus our strategy so far is to test the oldest and newest supported version. For simplicity, I'd recommend to stick with the existing strategy and update all tests to 10.2.

Reply:
Ok if you guys at Amazon agree, I guess it is fine for us! (actually I pinged you on slack @leezu )